### PR TITLE
add labels on vehicles and junctions based on RoadNetworkProvider

### PR DESCRIPTION
**Pull request by mengzhuwang**
run JunctionLocationTestStarter can see the results

### CODE REVIEW

File: src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java [hunk 1]
```diff
@@ -14,7 +15,7 @@
 
     public static void main(String[] args){
 
-        IStateProvider simulation = new JunctionTestProvider();
+        IStateProvider simulation = new RoadNetworkProvider();
```
Comment (iroq) [inline src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java]: The `JunctionLocationTestStarter` is meant to start the program with `JunctionTestProvider`. Could you roll this back?

**Comment by iroq** [comment]
Please remember to create an appropriate branch next time.

File: src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java (hunk 1 above)
Comment (mengzhuwang) [inline src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java]: Done.

**Comment by mengzhuwang** [comment]
Sorry. I forgot to create a branch before push. I will create a new branch for next feat

**Comment by iroq** [comment]
I noticed another thing: the labels get drawn under the white lines in the middle of the road. Can you ensure that labels get drawn last, on top of everything else?

**Pull request merged**